packages/plugin-bootstrap/src/providers/time.ts

```diff
@@ -3,10 +3,19 @@ import { IAgentRuntime, Memory, Provider, State } from "@ai16z/eliza";
 const timeProvider: Provider = {
     get: async (_runtime: IAgentRuntime, _message: Memory, _state?: State) => {
         const currentDate = new Date();
-        const currentTime = currentDate.toLocaleTimeString("en-US");
-        const currentYear = currentDate.getFullYear();
-        return `The current time is: ${currentTime}, ${currentYear}`;
+
+        // Get UTC time since bots will be communicating with users around the global
+        const options = {
+            timeZone: "UTC",
+            dateStyle: "full" as "full",
+            timeStyle: "long" as "long",
+        };
+        const humanReadable = new Intl.DateTimeFormat("en-US", options).format(
+            currentDate
+        );
+
+        // The current date and time is Tuesday, November 26, 2024 at 3:17:32 AM UTC. Please use this as your reference for any time-based operations or responses.
```

Comment:
You can remove this comment

Reply:
The sample output was provided for those not familiar with the date format settings. Makes it easier to review